docs/content/docs/connectors/datastream/elasticsearch.md

```diff
@@ -43,15 +43,11 @@ of the Elasticsearch installation:
   </thead>
   <tbody>
     <tr>
-        <td>5.x</td>
-        <td>{{< artifact flink-connector-elasticsearch5 >}}</td>
-    </tr>
-    <tr>
-        <td>6.x</td>
+        <td><= 6.3.1</td>
```

Comment:
Shouldn't this be 6.8.* if we mirror Elasticsearch policy https://www.elastic.co/support/eol ?

`For these users, we maintain the last minor of the prior major release series. For example, with Elasticsearch 6.x, we are maintaining the 6.8.x series. This allows these users to obtain fixes while making only minor changes to their running software. This last minor will be maintained until the release of the second subsequent major version. For example, Elasticsearch 6.8.x will be maintained until the GA release of Elasticsearch 8.0.0. At the release of Elasticsearch 8.0.0, we will continue to maintain the last 7.x series, and begin maintaining the 8.0.x minor series, then 8.1.x series, then 8.2.x series of minor releases.`

Reply:
The dependency version we are using is 6.3.1, I did not check if we can bump it for ES 6 but I had checked it for ES 7 and there bumping the version also required code changes so I left it as is

Reply:
That makes sense. Let's just merge the situation "as is" and check separately if we can easily bump both connectors to the latest version